plugin/template/README.md

```diff
@@ -0,0 +1,135 @@
+# template
+
+*template* allows for dynamic responses base on the incoming query.
+
+## Syntax
+
+~~~
+template CLASS TYPE [REGEX...] {
```

Comment:
what happens when multiple template match? First one wins?

Reply:
Added to the README